### PR TITLE
Update design-teams-app-overview.md

### PR DESCRIPTION
**Pull request by pallabiwrites**
The decsription of the "Collaborative" feature was not well-defined. It merely stated the information provided under the "Teams app design principles" heading. I have fixed that.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/81f19ab0850e0e646e7a79acb889aefee53d35e9/msteams-platform/concepts/design/design-teams-app-overview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8cbf6d74-bd57-4b67-a3df-4ae73d58d2b3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-abirade — CHANGES_REQUESTED** [review]
Request change

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We do not currently enforce a minimum score for content merged to the default branch. This will change in the future.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/design-teams-app-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc41b12c8d750be7484ab43ce65c4b4c1e031d7d/msteams-platform/concepts/design/design-teams-app-overview.md) | 93 | 7 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/31a6a3eb-40c3-4845-a594-9340d5790492) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/design/design-teams-app-overview.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Teams apps help people achieve more together. Use these principles to guide your
 
 ### Collaborative
 
-Teams apps help people achieve more together. Use these principles to guide your design.
+The app is designed to promote collaboration through coordinated and shared activities among the users.
```
[the remark this answers is not in the document]
Comment (pallabiwrites) [reply msteams-platform/concepts/design/design-teams-app-overview.md]: Fixed.

**Review by v-abirade — APPROVED** [review]
Approved

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**